app/questionnaire/questionnaire_store_updater.py

```diff
@@ -216,13 +216,23 @@ def update_answers(self, form_data, list_item_id=None):
         for answer_id, answer_value in form_data.items():
 
             if answer_id in answer_ids_for_question:
-                if answer_value not in self.EMPTY_ANSWER_VALUES:
+                answer_value_to_store = (
```

Comment:
Not 100% sure what this is doing, looks to me if answer_value is a dict, make another dict with all the empties removed. Then check it's not completely empty at a later stage?

Reply:
This is just filtering out any `EMPTY_ANSWER_VALUES` when the answer is a dict i.e address since we don't store any answers that match `EMPTY_ANSWER_VALUES`.